### ISSUE TITLE
Learn: logs from promises missing from fcc console

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/freeCodeCamp#38819]
These two challenge has the camper using a console.log, but nothing seems to get logged to the fCC console, only to the browser console.

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/handle-a-fulfilled-promise-with-then
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/handle-a-rejected-promise-with-catch

_Originally posted by @lasjorg in https://github.com/freeCodeCamp/freeCodeCamp/issues/38474#issuecomment-626261412_

**Comment by ojeytonwilliams** [comment]
Thanks for catching this @lasjorg, this is definitely not intended behaviour.

**Comment by Sembauke** [comment]
This seems to be fixed.

**Comment by ShaunSHamilton** [comment]
@Sembauke This is still an issue for me on Chrome. Which browser are you using?

**Comment by Sembauke** [comment]
Hey Shaun this works fine for me on Chrome version 90,
![image](https://user-images.githubusercontent.com/46919888/119254689-08be5500-bbb8-11eb-86b4-9e4083ad4f91.png)

**Comment by ShaunSHamilton** [comment]
Right, but the code you have is not what the lesson instructions ask for:
```javascript
const makeServerRequest = new Promise((resolve, reject) => {
  // responseFromServer is set to true to represent a successful response from a server
  let responseFromServer = true;
    
  if(responseFromServer) {
    resolve("We got the data");
  } else {  
    reject("Data not received");
  }
});
makeServerRequest.then(result => console.log(result));
```

**Comment by Nirajn2311** [comment]
Yup this issue is still there. Tested it in Mac.
